### PR TITLE
EISW-41342

### PR DESCRIPTION
**Pull request by weqaar**
### Details:
 - *modify openCV version and associated SHA to 4.5.5*
 - *modify cmake/developer_package/version.cmake for AIRA [EISW-41342]*
 
### Tickets:
 - *EISW-41342*

### CODE REVIEW

**Comment by ilyachur** [comment]
We don't have a plan to update 2022.1 release. Moreover please remove changes which are not related with current PR.

File: tools/mo/requirements.txt [hunk 1]
```diff
@@ -5,7 +5,7 @@ networkx~=2.5; python_version <= "3.6"
 networkx~=2.6; python_version > "3.6"
 numpy>=1.16.6,<1.20
 protobuf>=3.15.6
-onnx>=1.8.1
+onnx>=1.8.1,<1.12
```
Comment (rkazants) [inline tools/mo/requirements.txt:8]: Please explain why you did this in this PR? Why not `<=1.12`

**Comment by rkazants** [comment]
Could you please make more informative title for this PR? Now it is not too clear what this PR is about. Thanks!

**Comment by ilya-lavrenov** [comment]
@weqaar please, merge with latest `releases/2022/1.1`

**Comment by weqaar** [comment]
Cancelling this PR in lieu of releases/2022/1.1 rebased: https://github.com/openvinotoolkit/openvino/pull/12324

**Pull request closed without merging**